test/apps/iodemo/io_demo.cc

```diff
@@ -1882,8 +1888,13 @@ class DemoClient : public P2pDemoCommon {
 
             r->init(this, server_index, msg->sn, conn->id(), opts().validate,
                     iov, 0);
-            conn->recv_am_data((*iov)[0].buffer(), msg->data_size,
-                               (*iov)[0].memh(), data_desc, r);
+            if (!ucx_am_is_rndv(data_desc)) {
+                conn->recv_am_data(NULL, 0,
```

Comment:
wrap lines

Reply:
do you mean combining line 1892 & 1893 into one line?